module/behaviour/skills/DirectWalkController/src/DirectWalkController.cpp

```diff
@@ -85,11 +85,5 @@ namespace module::behaviour::skills {
                 emit(std::make_unique<ActionPriorities>(ActionPriorities{subsumptionId, {0, 0}}));
             }
         });
-
-        on<Trigger<WalkStopped>>().then([this] {
-            // TODO : Right now, this causes the walk engine to become unaware of positions and negatively
-            // impact servo positions... emit(std::make_unique<ActionPriorities>(ActionPriorities {
-            // subsumptionId, { 0, 0 }}));
-        });
```

Comment:
I feel like this should remain with a more pointed TODO message. Unless there are no actions that need to be taken by the walk controllers (this should be compared with the other walk controllers) when the walk stops?

Reply:
That TODO refers directly to the oldwalkengine (it was written 4 years ago), and there aren't any other triggers in the codebase for `WalkStopped`

Reply:
That's wrong. other controllers do use it. I had files in the `files to include` in my search :facepalm:

Reply:
[`SimpleWalkPathPlanner`](https://github.com/NUbots/NUbots/blob/master/module/behaviour/planning/SimpleWalkPathPlanner/src/SimpleWalkPathPlanner.cpp#L123) has a trigger for it where it emits a new `ActionPriorities` which just includes the `subsumptionId`